qa-tests-backend/src/main/groovy/services/NetworkGraphService.groovy

```diff
@@ -92,6 +94,10 @@ class NetworkGraphService extends BaseService {
                         GetExternalNetworkEntitiesRequest.newBuilder().setClusterId(clusterId).build()
                 GetExternalNetworkEntitiesResponse response =
                         getNetworkGraphClient().getExternalNetworkEntities(request)
+
+                // Potential of causing io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED
+                log.info "getExternalNetworkEntities response: ${response}"
+                log.info "getExternalNetworkEntities allEntities: ${response.getEntitiesList()}"
```

Comment:
Are you keeping this after this PR? Because if the payload is larger than 4MB then I assume this will add many lines to the logs.

Also, I checked the successful runs of QA tests and didn't find occurrences of this log. Are we sure this is running?

Reply:
Yes, it is running, the full logs are 170MB long thanks to this :) 

No, I think it should not be merged here. I will split that part into a separate PR where we can investigate why the reply is so enormous. Sorry, I should have split that before requesting reviews.

Reply:
Could you link the logs where these lines are shown? I couldn't find them. Curious to see whats there. I guess we could leave a log line with a count perhaps (i.e. length of `response.getEntitiesList()`)?

Reply:
```
curl -sSL -o logs https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/stackrox_stackrox/8974/pull-ci-stackrox-stackrox-master-gke-qa-e2e-tests/1733217666161709056/artifacts/gke-qa-e2e-tests/stackrox-stackrox-e2e-test/build-log.txt
```


I am betting, the `response.getEntitiesList()` is 1. But it includes many `info` objects inside. You will see that when you open the log file.